gradlew

```diff
@@ -1,8 +1,19 @@
 #!/bin/sh
 
 #
-# Copyright (c) 2020-2023 Tesla (Yinsen) Zhang.
-# Use of this source code is governed by the MIT license that can be found in the LICENSE.md file.
+# Copyright © 2015-2021 the original authors.
```

Comment:
Of course this file is copied from the original author.

Reply:
Sure